### DISCUSSION TITLE
Running OpenDTU offline

### DISCUSSION THREAD

**Discussion by fischer-felix** [tbnobody/OpenDTU#601] [Q&A]
Hi, we would like to set up a small solar system at my grandma's house, however she does not have internet at her house.

From reading the documentation it seems like it is currently not possible to keep the esp's wifi hotspot on and set the time manually.

Is this correct and if so, is there any way around this?

**Comment by Solarteur** [comment]
Regardless of whether the AP mode can also be permanently available, the following note from "NTP Settings" is important:
![Hint](https://user-images.githubusercontent.com/117977338/220126495-d5285c3d-01b2-44fd-9a1a-c9dd3d5893b7.png)

**Comment by huckes** [comment]
Just an idea: Take an ESP8266 to span a local WLAN, connect a DCF-receiver to the ESP and connect the OpenDTU via WLAN to the ESP.
As starting point you might like to take a look on EddyVB's project: https://github.com/EddyVB/ESP8266-NTP2DCF77. 

Or post an PR to this OpenDTU-project, the ESP32 might have enough resources to get an DCF-receiver connected. That were the very smart way, but someone has to implement this enhancement.
